docs/tutorials/automatic-differentiation.md

```diff
@@ -1,8 +1,210 @@
-# Automatic differentiation
+# Automatic differentiation 101
 
-```{note}
-This is a placeholder for a section in the new {ref}`jax-tutorials`.
+JAX has a pretty general automatic differentiation (autodiff) system. Computing gradients is a critical part of modern machine learning methods, and this tutorial will walk you through a few introductory autodiff topics.
 
-For the time being, you may find some related content in the old documentation:
-- {doc}`../jax-101/04-advanced-autodiff`
-```
\ No newline at end of file
+While understanding how automatic differentiation works "under the hood" isn't crucial for using JAX in most contexts, you are encouraged to check out this quite accessible [video](https://www.youtube.com/watch?v=wG_nF1awSSY) to get a deeper sense of what's going on.
+
+And if you are looking for more advanced material, refer to the [Advanced autodiff](https://jax.readthedocs.io/en/latest/tutorials/advanced_autodiff.html) tutorial.
+
+## Gradients
+
+In JAX, you can differentiate a function with the `jax.grad` transform:
+
+```python
```

Comment:
Probably use `code-cell` here so the output will be embedded. This will require adding jupytext info to the header; you can copy and paste from `quickstart` or one of the other already-completed tutorials.

Same comment for other code blocks below.